src/app_service/service/wallet_account/service.nim

```diff
@@ -78,7 +78,8 @@ type DerivedAddressesArgs* = ref object of Args
 type TokensPerAccountArgs* = ref object of Args
   accountsTokens*: OrderedTable[string, seq[WalletTokenDto]] # [wallet address, list of tokens]
 
-const CheckBalanceIntervalInMilliseconds = 15 * 60 * 1000 # 15 mins
+const CheckBalanceSlotExecuteIntervalInMilliseconds = 15 * 60 * 1000 # 15 mins
+const CheckBalanceTimerIntervalInMilliseconds = 1000 # 1 sec
```

Comment:
@endulab for this particular issue we cannot set in any case interval to 1 sec, that is too often check and will increase a lot infura calls. So that's not an option at all, please revert it to 15 mins. Have you tried to kill a thread from a destructor of this service (`delete` proc)?

If there is no an easy way to do that then we need to extend threading mechanism where it will return id of a certain thread when we're submitting a task for execution in a separate thread, and once the app is about to close to fetch that tread via id and kill it before app closing.

Reply:
@saledjenic the Infura is not called every second. It is called every 15 minutes. Please read my explanation in PR description.

Reply:
@endulab yes, I see what was your idea, kind of a trick. Have you looked for a more appropriate solution (maybe something what I suggested)?

Reply:
@saledjenic I created a separated issue for investigating possible solutions: #6179
Unfortunately there is no delete proc for Thread class. I tried to do this in threadpool. Also destroyThread() (which uses pthread) is not available.
Let's merge current solutions since it causes a lot of problems and I will do some research in #6179